velox/functions/prestosql/DecimalFunctions.cpp

```diff
@@ -163,6 +163,43 @@ struct DecimalDivideFunction {
   uint8_t aRescale_;
 };
 
+template <typename TExec>
+struct DecimalModulusFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(TExec);
+
+  template <typename A, typename B>
+  void initialize(
+      const std::vector<TypePtr>& inputTypes,
+      const core::QueryConfig& /*config*/,
+      A* /*a*/,
+      B* /*b*/) {
+    auto aType = inputTypes[0];
+    auto bType = inputTypes[1];
+    auto [aPrecision, aScale] = getDecimalPrecisionScale(*aType);
+    auto [bPrecision, bScale] = getDecimalPrecisionScale(*bType);
+    aRescale_ = computeRescaleFactor(aScale, bScale);
+    bRescale_ = computeRescaleFactor(bScale, aScale);
+  }
+
+  template <typename Q, typename A, typename B>
+  void call(Q& out, const A& a, const B& b) {
+    Q quotient;
+    out = DecimalUtil::divideWithRoundUp<Q, A, B>(
```

Comment:
It seems wasteful to use divideWithRoundUp which produces both remainder and quotient. Can we introduce a different helper function that only returns the remainder? It is also strange to call a function named divideWithRoundUp and pass roundUp flag 'false'?